jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -214,13 +222,15 @@ public static void setServerInfo(String serverInfo)
     private final Set<EventListener> _durableListeners = new HashSet<>();
     private String[] _protectedTargets;
     private final CopyOnWriteArrayList<AliasCheck> _aliasChecks = new CopyOnWriteArrayList<>();
-    private Availability _availability = Availability.UNAVAILABLE;
 
     public enum Availability
     {
         UNAVAILABLE, STARTING, AVAILABLE, SHUTDOWN,
     }
 
+    ;
```

Comment:
:confused: :grey_question:

Reply:
Always been there, and asked to not do other changes that were not related to synchronization.
I may still have slipped a few though.